src/transformers/pipelines/text2text_generation.py

```diff
@@ -273,7 +273,7 @@ def check_inputs(self, input_length: int, min_length: int, max_length: int) -> b
 
         if input_length < max_length:
             logger.warning(
-                f"Your max_length is set to {max_length}, but your input_length is only {input_length}. You might "
+                f"Your max_length is set to {max_length}, but your input_length is only {input_length}. Since this is a summarization task, where outputs shorter than the input are typically wanted, you might "
```

Comment:
Can you just respect the 119 char length here (not sure why black didn't do it).

Reply:
Interesting, I relied on `black` to fix it for me :D 

pushing a fix